src/data/customer/Driver.js

```diff
@@ -115,6 +116,95 @@ export const createDriverAsync = async (
   }
 };
 
+/**
+ * Create Driver
+ * transactionGroupIds (mandatory) (array)
+ * sendConfirmationSms (optional) (boolean)
+ * sendConfirmationEmail (optional) (boolean)
+ * driverTypeIds (mandatory) (array) #csv Eg. [2,3]
+ * firstName (mandatory) (string)
+ * lastName (mandatory) (string)
+ * email (mandatory) (string)
+ * password (mandatory) (string)
+ * birthday (mandatory) (string) (Y-m-d)
+ * phone (mandatory) (string)
+ * vehicle (array) contains vehicle info
+ * vehicle.vehicleColor (optional) (string)
+ * vehicle.vehicleAverageSpeed (int) (string)
+ * vehicle.vehicleMaximumCapacity (int) (string)
+ * vehicle.vehicleModelYear (optional) (int)
+ * vehicle.vehicleLicenseNumber (optional) (int)
+ * vehicle.vehicleBrand (optional) (string)
+ * vehicle.vehicleModel (optional) (string)
+ * vehicle.vehicleTypeId (optional) (int)
+ * @param {object} driverInfo {}
+ * @param {string} token
+ * @return {object} Promise resolve/reject
+ * deprecated since verion 0.1.77
+ */
+export const createDriverAsync = async (
+  {
+    transactionGroupIds,
+    sendConfirmationSms = false,
+    sendConfirmationEmail = false,
+    driverTypeIds,
+    firstName,
+    lastName,
+    email,
+    password,
+    birthday,
+    phone,
+    vehicleColor,
+    vehicleAverageSpeed,
+    vehicleMaximumCapacity,
+    vehicleModelYear,
+    vehicleLicenseNumber,
+    vehicleBrand,
+    vehicleModel,
+    vehicleTypeId,
+  },
+  token,
+) => {
+  try {
+    const data = {
+      transactionGroupIds,
+      sendConfirmationSms,
+      sendConfirmationEmail,
+      driverTypeIds,
+      firstName,
+      lastName,
+      email,
+      password,
+      birthday,
+      phone,
+      vehicle: {
+        vehicleColor,
+        vehicleAverageSpeed,
+        vehicleMaximumCapacity,
+        vehicleModelYear,
+        vehicleLicenseNumber,
+        vehicleBrand,
+        vehicleModel,
+        vehicleTypeId,
+      },
+    };
+
+    const response = await axios({
+      method: 'POST',
+      url: endpoints.API_V3.CUSTOMER_DRIVER,
+      headers: {
+        Authorization: `Bearer ${token}`,
+        'Content-Type': 'application/json',
+      },
+      data,
```

Comment:
@vdanylov you likely need to convert to snake_case, please check other functions and API documentation